07.ls_object/lib/screen.rb

```diff
@@ -22,21 +22,21 @@ def show
     output_blocks << create_output_block_with_file_stats
     output_blocks << create_output_blocks_with_dir_stats
 
-    output_blocks.reject(&:empty?).join("\n\n").encode('UTF-8')
+    output_blocks.reject(&:empty?).join("\n\n")
   end
 
   private
 
   def create_output_block_with_file_stats
-    formatter_factory.new(@stats['']).write
+    formatter_factory.new(@ls_file_stats['']).write
   end
 
   def create_output_blocks_with_dir_stats
-    @stats.except('').map do |dir_path, stat|
+    @ls_file_stats.except('').map do |dir_path, stats|
```

Comment:
```suggestion
    @ls_file_stats_by_dir.except('').map do |dir, ls_file_stats|
```

ディレクトリ別にLsFileStatの配列が格納されているHash、というのを表現するのなら、やっぱり `ls_file_stats_by_dir` がいいかもしれませんね。
ちょっと長いのが気になりますが、変数の中身を適切に表現しているという点ではこの方がいいのかな、と思いました